src/main/java/com/pst/user/service/UserService.java

```diff
@@ -1,32 +1,121 @@
 package com.pst.user.service;
 
+import java.util.ArrayList;
+
 import java.util.List;
 import java.util.Optional;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import com.pst.user.entity.UserEntity;
-import com.pst.user.repository.UserRepo;
+import com.pst.user.repository.UserRepository;
+import com.pst.user.response.UserResponse;
 
 @Service
 public class UserService {
 
 	@Autowired
-	UserRepo repositary;
-
-	public List<UserEntity> getAllUsers() {
-		return repositary.findAll();
+	private UserRepository userRepositary;
+    @Autowired
+	private UserResponse userResponse;
```

Comment:
create object for UserResponse
Suggestion : private UserResponse userResponse = new UserResponse();